src/app.js

```diff
@@ -171,16 +171,17 @@ async function getVideoData () {
         error(`failed to parse the lesson page '${urlValue}'}`)
       }
     } else {
-      let lessonURLs = []
       success('The URL is a playlist or series')
 
       // get the urls of the lessons
-      const re = /<a class="flex h-100 absolute top-0 right-0 bottom-0 left-0" style="pointer-events:all;" href="(\/lessons\/.+?)"/g
+      const re = /<a .* href="(\/lessons\/.+?)"/g
```

Comment:
It will select all videos, including in featured section. You need to change it and make it unique to course videos.

Reply:
It doesn't do it because I think featured section isn't loaded from the beginning